src/main/java/com/isa/morswiny/users/User.java

```diff
@@ -20,18 +20,21 @@
     private String login;
     private String email;
     private String password;
-    private LocalDateTime birthday;
```

Comment:
zmien prosze ID na id - wiem, ze to nie Twoj kod, ale moglbys od razu podmienic sprytnie :-)